components/consistent-evaluation.js

```diff
@@ -48,14 +56,21 @@ export class ConsistentEvaluation extends MobxLitElement {
 			this._childHrefs = await controller.getHrefs();
 			this._submissionInfo = await controller.getSubmissionInfo();
 			this._gradeItemInfo = await controller.getGradeItemInfo();
+			this._assignmentName = await controller.getAssignmentName();
+			this._organizationName = await controller.getOrganizationName();
+			this._iteratorTotal = await controller.getIteratorTotal();
+			this._iteratorIndex = await controller.getIteratorIndex();
 		}
-
 	}
 
 	onNextStudentClick() {
 		this.href = this._childHrefs.nextHref;
 	}
 
+	onPreviousStudentClick() {
+		this.href = this._childHrefs.previousHref;
```

Comment:
```suggestion
		this.href = this._childHrefs?.previousHref;
```
Suggest a `?` just in case `_childHrefs` were to be undefined.

Reply:
Could this also apply to the `onNextStudentClick()` as well?